zone/perl_mob.cpp

```diff
@@ -3636,12 +3636,12 @@ XS(XS_Mob_SetHate); /* prototype to pass -Wmissing-prototypes */
 XS(XS_Mob_SetHate) {
 	dXSARGS;
 	if (items < 2 || items > 4)
-		Perl_croak(aTHX_ "Usage: Mob::SetHate(THIS, Mob* other, [int32 hate = 0], [int32 damage = 0])"); // @categories Hate and Aggro
+		Perl_croak(aTHX_ "Usage: Mob::SetHate(THIS, Mob* other, [int32 hate = 0], [int64 damage = 0])"); // @categories Hate and Aggro
 	{
 		Mob *THIS;
 		Mob *other;
 		int32 hate;
-		int32 damage;
+		int64 damage;
```

Comment:
Hate, too? Still casting to int32 for both below as well.

Reply:
Definitely not done yet hence the draft, still plan on making sure all the edges are touched

Reply:
No worries. As far as I'm aware, int64 does not function with the Perl API. See $client->GetCarriedMoney() with over 2,000,000 Platinum for example.

Reply:
Perl should have support given 64 bit installation / libs

https://stackoverflow.com/questions/31490111/using-64-bits-integers-with-perl-xs

That's not to say we might need to do additional investigation